pkg/controllers/openshift-controller-manager.go

```diff
@@ -56,58 +57,50 @@ func (s *OCPControllerManager) Dependencies() []string {
 
 func (s *OCPControllerManager) configure(cfg *config.MicroshiftConfig) {
 	s.kubeconfig = cfg.KubeConfigPath(config.KubeAdmin)
+	servingCertDir := cryptomaterial.OpenshiftControllerManagerServingCertDir(cryptomaterial.CertsDirectory(cfg.DataDir))
 
-	if err := s.writeConfig(cfg); err != nil {
-		klog.Fatalf("Failed to write openshift-controller-manager config %v", err)
+	ocmconfig := openshiftcontrolplanev1.OpenShiftControllerManagerConfig{
+		KubeClientConfig: configv1.KubeClientConfig{
+			KubeConfig: s.kubeconfig,
+		},
+		Controllers: []string{
+			"*",
+			"-openshift.io/build",
+			"-openshift.io/build-config-change",
+			"-openshift.io/default-rolebindings",
+			"-openshift.io/deployer",
+			"-openshift.io/deploymentconfig",
+			"-openshift.io/image-import",
+			"-openshift.io/image-signature-import",
+			"-openshift.io/image-trigger",
+			"-openshift.io/origin-namespace",
+			"-openshift.io/serviceaccount",
+			"-openshift.io/serviceaccount-pull-secrets",
+			"-openshift.io/templateinstance",
+			"-openshift.io/templateinstancefinalizer",
+			"-openshift.io/unidling",
+		},
+		LeaderElection: configv1.LeaderElection{
+			Disable:       true,
```

Comment:
this is not going to work now since we would need to use a builder in the controller to utilize this, but it will be possible once we refactor route-controller-manager to BuilderController